test/fixtures/user-agent-strings.js

```diff
@@ -0,0 +1,463 @@
+export default {
+    Safari: {
+        '11': {
+            isIDBValid: true,
+            userAgentStrings: [
+                'Mozilla/5.0 (iPad; CPU OS 11_0 like Mac OS X) AppleWebKit/604.1.25 (KHTML, like Gecko) Version/11.0 Mobile/15A5304j Safari/604.1'
+            ]
+        },
+        '10.1.2': {
+            isIDBValid: true,
+            userAgentStrings: [
+                'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/603.3.8 (KHTML, like Gecko) Version/10.1.2 Safari/603.3.8'
+            ]
+        },
+        '10.1.1': {
+            isIDBValid: true,
```

Comment:
I might be missing something, but these are just our test case expectations and thought that I shouldn't have ANY logic in there.
I agree that we could do `isIDBValid = semver.gte(version, '10.2.0')` but this would duplicate our validity condition in the test cases as well.
If you thing that we should change that, plz clarify the suggested implementation and will update the code :)

Reply:
Maybe we could have two objects... one for the valid versions and one for the invalid.
Sound a bit DRYer. What do you think?

Reply:
Ah actually fair point. Two separate objects is nicer, yeah. That will do the trick 👍